### PR TITLE
Email preferences: Long Reads

### PR DESCRIPTION
**Pull request by rrees**
Adds the new Long Reads email to the preferences centre

### CODE REVIEW

File: identity/app/model/EmailSubscriptions.scala [hunk 1]
```diff
@@ -118,6 +118,19 @@ object EmailSubscriptions {
...
+  def newsEmails(subscribedListIds: Iterable[String] = None) = List(
+    EmailSubscription(
+      "The Long Read",
+      "news",
+      "The week’s Long Reads and audio features",
+      "Bringing you the latest Long Read features and podcasts, delivered to your inbox every Saturday morning",
+      "Every week",
+      "3322",
+      0,
+      subscribedTo = subscribedListIds.exists{ x => x == "3322" }
```
Comment (rich-nguyen) [inline identity/app/model/EmailSubscriptions.scala]: `contains`

**Comment by rich-nguyen** [comment]
:+1:

File: identity/app/model/EmailSubscriptions.scala (hunk 1 above)
Comment (rich-nguyen) [inline identity/app/model/EmailSubscriptions.scala]: oh, its an `Iterable`

Comment (rrees) [inline identity/app/model/EmailSubscriptions.scala]: Yeah, I suspect the type is a List somewhere higher up in the code.

**Pull request merged**